src/Core/Models/Business/SecretsManagerSubscriptionUpdate.cs

```diff
@@ -6,16 +6,15 @@ namespace Bit.Core.Models.Business;
 
 public class SecretsManagerSubscriptionUpdate
```

Comment:
Since we already have the methods `AdjustSeats` and `AdjustServiceAccounts` what do you think of adding methods `SetMaxAutoscaleSmSeats` and `SetMaxAutoscaleSmServiceAccounts` and then make the properties have `private set`?
I just think it looks a bit weird to use methods to set a few properties and then use the object initializer to set other properties.

Reply:
I don't really see any issue with it to be honest. We could move to all method setters, but there are times we also want to set seats and service accounts directly, so we'd end up with `SetSmSeats`, `SetSmServiceAccounts`, `SetMaxAutoscaleSmSeats` and `SetMaxAutoscaleSmServiceAccounts`, as well as the 2 `Adjust` methods. (Plus proration date?)

I thought that would be unnecessary boilerplate when we can just set those in the initializer. Whereas the `Adjust` methods do actually encapsulate some additional logic.

Reply:
Fair enough, that is a good point. Thanks for considering it.